<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRoomDialog</class>
 <widget class="QDialog" name="CreateRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="cbPassword">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lePassword">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="leRoomname"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QSpinBox" name="sbLimit">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
